controllers/dscinitialization/servicemesh_setup.go

```diff
@@ -77,8 +78,9 @@ func configureServiceMeshFeatures() feature.FeaturesProvider {
 				PreConditions(
 					servicemesh.EnsureServiceMeshInstalled,
 				).
+				ManifestSource(dsciEmbeddedFS).
 				Manifests(
-					path.Join(feature.ServiceMeshDir, "metrics-collection"),
+					path.Join(metricsDir),
```

Comment:
maybe i missed something, just try to understand why some use dir, but others list all files:
say under "metricsDir" 2 files,  under "meshDir" one file
but line 60 points to that template yaml , but here is use the folder.
should line 60 not be 
`path.Join(meshDir),`
than 
`path.Join(meshDir, "create-smcp.tmpl.yaml"),`

Reply:
+1 
1099d0a